CSharp/BatchExplorer/Models/TaskModel.cs

```diff
@@ -279,6 +267,18 @@ public override List<PropertyModel> PropertyModel
                         //Swallow the exception to stop popups from occuring for every bad VM
                     }
 
+                    IPagedEnumerable<SubtaskInformation> subtasks = this.Task.ListSubtasks(OptionsModel.Instance.ListDetailLevel);
```

Comment:
Should this be in a try-swallow block to "stop popups occurring for every bad VM" too?